client/components/Application/LowerMenu.tsx

```diff
@@ -0,0 +1,40 @@
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import reportIssueIcon from '../../assets/report_issue_icon.svg'
+import userGuideIcon from '../../assets/user_guide_icon.svg'
+
+export default function LowerMenu() {
```

Comment:
@guergana have you tried https://mui.com/material-ui/react-list/? Looks like a ready to go component that could save us from creating our custom one.

I don't have any particular comment regarding your implementation, I'm just wandering if you tried it and why did you chose for a custom one.



I

Reply:
@pdelboca I didn't know about this. this looks like the appropriate component but I dont know if we should stick to MUI Components in this case. I think the example implementation has too much overhead. I used `Box` and `Button` and to implement a List in MUI i need 5 components: `List`, `ListItem`, `ListItemButton`, `ListItemIcon`, `ListItemText`. It might be better for semantic purposes though and maybe the accessibility is better but to me it looks too cluttered for two simple buttons. What do you think??